libr/core/anal.c

```diff
@@ -1445,32 +1342,29 @@ R_API int r_core_anal_graph(RCore *core, ut64 addr, int opts) {
 			" fontname=\"%s\" fontsize=\"8\"];\n", font);
 	if (is_json)
 		r_cons_printf ("[");
-	//r_cons_flush ();
-#define inrange(x,f) ((x>=f->addr)&&(x<(f->addr+f->size)))
 	r_list_foreach (core->anal->fcns, iter, fcni) {
 		if (fcni->type & (R_ANAL_FCN_TYPE_SYM | R_ANAL_FCN_TYPE_FCN)
-				&& (addr == 0 || inrange (addr, fcni))) {
```

Comment:
We have fcn_at and fcn_in functions to clarify this. Those macros should use the same logic (need more context and a keyboard to make a stronger comment on this)



> On 03 Aug 2015, at 21:08, Jeffrey Crowell <notifications@github.com> wrote:
> 
> In libr/core/anal.c:
> 
> >  	r_list_foreach (core->anal->fcns, iter, fcni) {
> >  		if (fcni->type & (R_ANAL_FCN_TYPE_SYM | R_ANAL_FCN_TYPE_FCN)
> > -				&& (addr == 0 || inrange (addr, fcni))) {
> is infun the same as inrange?
> 
> —
> Reply to this email directly or view it on GitHub.
>